src/convert_image/sse2.rs

```diff
@@ -221,6 +223,30 @@ unsafe fn unpack_ui8x3_i16x2_4x(image: *const u8, sampler: Sampler) -> (__m128i,
     (_mm_or_si128(red, green), _mm_or_si128(blue, green))
 }
 
+#[inline(always)]
+unsafe fn rgb_to_bgra_lane_conversion(input: __m128i, output_buffer: *mut __m128i) {
+    let alpha_mask = _mm_set_epi32(0xff, 0xff, 0xff, 0xff);
```

Comment:
Since the pattern is repeating, you can use `_mm_set1_epi32(DEFAULT_ALPHA as i32)`.